Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
+         if File.directory?(f)
+           false
+         else
+           f = File.basename(f)
+           !(Metafiles.copy?(f) || %w[.DS_Store INSTALL_RECEIPT.json].include?(f))
+         end
+      end
+      problem <<-EOS.undent
+        The installation seems to be empty. Please ensure the prefix
+           is set correctly and expected files are installed.
```

Comment:
It looked incredibly strange on my Terminal without the lines indented. Potentially just a weird thing with my terminal though, I'll nudge it back in line.